File: apps/problems/simple-array/boilerplate-full/function.rs

use std::io::{self, BufRead};

##USER_CODE_HERE##

fn main() {
  let stdin = io::stdin();
  let mut input = stdin.lock().lines().map(|l| l.unwrap());
  let num1: i32 = input.next().unwrap().parse().unwrap();
  let num2: i32 = input.next().unwrap().parse().unwrap();
  let result = sum(num1, num2);
  println!("{}", result);
}
    

File: apps/problems/simple-array/boilerplate-full/function.js

##USER_CODE_HERE##

const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
const num1 = parseInt(input.shift());
  const num2 = parseInt(input.shift());
const result = sum(num1, num2);
console.log(result);
    